### ISSUE TITLE
Assert Count command wrong result with minimum and maximum provided

### ISSUE THREAD

**Issue by k-butz** [reubenmiller/go-c8y-cli#291]
The `c8y measurements assert count` command seems to treat minimum and maximum as OR instead of AND. Steps to reproduce:
1. Create a new Device
```sh
-> c8y devices create --name test -f --select id,name -o json -c
{"id":"12162634012","name":"test"}
```
2. Prove that there are 0 measurements: 
```sh
-> c8y measurements list --device 12162634012 --withTotalElements --withTotalPages -o json -c
{"currentPage":1,"pageSize":5,"totalElements":0,"totalPages":0}
```
3. Execute assert count command with minimum 1 and maximum 10:
```
-> c8y measurements assert count --minimum 1 --maximum 10 --device 12162634012
12162634012
```

Expected: The Device has 0 Measurements, assert count should not pass device id to output
Actual: The command see's the condition as fulfilled and lets the Device ID pass

**Comment by reubenmiller** [comment]
I can confirm it definitely is a bug in all "count" related assertions (measurements, alarms, events, operations etc.). There is a simple logical bug when the maximum is greater than -1, then it will ignore the previous minimum assertion.

Snippet from: pkg/c8ywaiter/alarms.go

```go
                 if s.Minimum > -1 {
			done = count >= int(s.Minimum)
		}

		if s.Maximum > -1 {
			done = count <= int(s.Maximum)
		}
```

This should be a really easy/quick fix. Apologies for the dumb logical error. Thanks for reporting it

**Comment by reubenmiller** [comment]
Resolved in [2.37.0](https://github.com/reubenmiller/go-c8y-cli/releases/tag/v2.37.0)
